app/src/main/java/com/example/munchbox/ui/MealSelectionScreen.kt

```diff
@@ -42,6 +44,8 @@ fun MealSelectionScreen(
     /**
      * State Variables
      */
+    val orderService = StorageServices(FirebaseFirestore.getInstance()).orderService()
```

Comment:
@Seanlebon this is what i mean by having to get the instance again

Reply:
We can probably pass it down once we set up everything else to be able to take in the service (aka once we restructure the code to user the DB)